src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -794,12 +802,20 @@ func (s *service) fetchReadEncoded(ctx context.Context,
 	defer sp.Finish()
 
 	i := 0
+	reader := docs.NewEncodedDocumentReader()
 	for _, entry := range results.Map().Iter() {
 		idx := i
 		i++
 
-		tsID := entry.Key()
-		tags := entry.Value()
+		tsID := ident.BytesID(entry.Key())
```

Comment:
We probably want to use a single `ident.NewReuseableBytesID()` and a call to `id.Reset(entry.Key())` to avoid the allocation of a cast to `BytesID` for each key.